diff --git a/added.txt b/added.txt
new file mode 100644
index 0000000..536f23d
--- /dev/null
+++ b/added.txt
@@ -0,0 +1 @@
++++ content
diff --git a/binary.dat b/binary.dat
index e7be1ea..f9e371f 100644
Binary files a/binary.dat and b/binary.dat differ
diff --git a/deleted.txt b/deleted.txt
deleted file mode 100644
index 71779d2..0000000
--- a/deleted.txt
+++ /dev/null
@@ -1 +0,0 @@
-deleted
diff --git a/mode.sh b/mode.sh
old mode 100644
new mode 100755
diff --git a/old name.txt b/new name.txt
similarity index 100%
rename from old name.txt
rename to new name.txt
diff --git a/nonewline.txt b/nonewline.txt
index 489ce0f..3e5126c 100644
--- a/nonewline.txt
+++ b/nonewline.txt
@@ -1 +1 @@
-old
\ No newline at end of file
+new
\ No newline at end of file
diff --git "a/quote\tname.txt" "b/quote\tname.txt"
index 3367afd..3e75765 100644
--- "a/quote\tname.txt"
+++ "b/quote\tname.txt"
@@ -1 +1 @@
-old
+new
diff --git a/space name.txt b/space name.txt
index 4cb29ea..d12f96b 100644
--- a/space name.txt	
+++ b/space name.txt	
@@ -1,3 +1,3 @@
 one
-two
+<script>alert(1)</script>
 three
